h5p-bildetema-topic-list/tsconfig.json

```
@@ -16,5 +16,5 @@
     "resolveJsonModule": true,
     "skipLibCheck": true
   },
-  "include": ["./src/**/*", "typings.d.ts"]
+  "include": ["./src/**/*", "typings.d.ts", "../common/utils/data.utils.ts"]
```

We might want to move all code into a `src` directory within `common`, and then we can do this:

```suggestion
  "include": ["./src/**/*", "typings.d.ts", "../common/src/**/*"]
```